### ISSUE TITLE
Level 18 (nth child): unclear explanation

### ISSUE THREAD

**Issue by charlesdevops** [flukeout/restaurant#122]
The only example is:

"div p:nth-child(2) selects the second p in every div"

So it's easy to take out the notion that we HAVE to specify the parent element. Nonetheless, the "div plate:nth-child(3)" is not even recognized as correct, while "plate:nth-child(3)" works.

**Comment by adam-beck** [comment]
Yes this had me confused for a while. It definitely is valid for anybody coming here for help/understanding.

https://jsfiddle.net/n5qy6sz6/

**Comment by adam-beck** [comment]
And just for the record (and maybe this should be a separate issue since this one is specific) level 19 has the same problem.

**Comment by Droberts359** [comment]
Just clarifying. This is actually the correct syntax? 

div plate:nth-child(3)

**Comment by StringHead** [comment]
> 
> 
> Just clarifying. This is actually the correct syntax?
> 
> div plate:nth-child(3)

Yes, it is. Sollution still doesn't work...

**Comment by yonderbread** [comment]
Having issue with this too

**Comment by luckyhat4** [comment]
Came here to say I was stuck on this, didn't even consider trying to do it without specifying the parent.
div plate:nth-child(3) should work regardless.

**Comment by hnm938** [comment]
div plate:nth-child(3) doesn't work but you can use plate:nth-of-type(3)

**Comment by NateRowsell** [comment]
Came here for the same thing div plate:nth-child(3) still does not work. Had me baffled for a little while.

**Comment by mark-aldrin-b** [comment]
:nth-child(3)

**Comment by ismailcakmak** [comment]
issue still persist in 2025, crazy
